### ISSUE TITLE
PlaceSpotOrder can't parse decimal numbers.

### ISSUE THREAD

**Issue by Trumpeta2021** [burakoner/BitMax.Net#8]
It is not possible to use decimal numbers in PlaceSpotOrder. Your examples are not working as well. It says Error (300001) Unable to parse order price - invalid number format.
Using integers is fine.

**Comment by burakoner** [comment]
I'm working on a big update including this error. I think it'll be finished tonight or tomorrow. Your Windows Language causes this problem. I can offer a temporary solution until next release. You can change CurrentCulture of Thread as below.

```
Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
Thread.CurrentThread.CurrentUICulture = CultureInfo.InvariantCulture;
```

Have a nice day.

**Comment by burakoner** [comment]
You can try now with https://github.com/burakoner/BitMax.Net/commit/a59b6019ea54db4f600b72a26e89ccd59ccefdd3 or https://www.nuget.org/packages/BitMax.Net/2.0.0

**Comment by Trumpeta2021** [comment]
Perfect. Thank you for your great work! I temporarily changed CurrentCulture and now it works.

**Comment by burakoner** [comment]
I've released version 2.0.0. You may want to upgrade
